index.js

```diff
@@ -89,3 +89,6 @@ globals.setImmediate(function () {
 });
 
 module.exports = runner.test;
+// TypeScript imports the `default` property for
+// an ES2015 default import (`import test from 'ava'`)
```

Comment:
Might be useful to link to a background explanation for this. I found <https://github.com/Microsoft/TypeScript/issues/2242#issuecomment-83694181>, not sure if there's official documentation for this behavior elsewhere?

tl;dr TypeScript's behavior doesn't seem without merit, and explicitly exporting a `default` here should be okay.